README.md

```diff
@@ -40,6 +40,35 @@ val client = S3AsyncClient
               .region(Region.EU_CENTRAL_1)
               .httpClient(akkaClient)
               .build()
+
+val eventualResponse = client.listBuckets()
+```
+
+If you need to connect to some AWS service from internal Corporate, where is needed to configure a Proxy, it can be performed in the following way:
+
+```scala
+val proxyHost = "localhost"
```

Comment:
Thanks for adding this code example to the documentation!!! 
Can you please add `val system = ActorSystem("aws-akka-http")` above proxyHost? This is necessary to use the `system` down in the client setup.